config.travis.py

```diff
@@ -20,24 +19,31 @@
     'channel': os.getenv("TRAVIS_SLACK_CHANNEL_ID")
 }
 
+
 class GOOGLE_CAL:
+    CAL_ID = os.getenv("TRAVIS_GOOGLE_CAL_ID", "")
+    CAL_API_KEY = os.getenv("TRAVIS_GOOGLE_CAL_API_KEY", "")
+
+    # optional - only needed if private data needs to be accessed
     SCOPES = ["https://www.googleapis.com/auth/calendar.readonly"]
```

Comment:
same thing here, these optional keys aren't referenced so they should be removed if we don't plan on using them